products/memcache/api.yaml

```diff
@@ -118,6 +153,16 @@ objects:
         description: |
           Number of nodes in the memcache instance.
         required: true
+      - !ruby/object:Api::Type::Enum
+        name: memcacheVersion
+        description: |
+          The major version of Memcached software. If not provided, latest supported version will be used.
+          Currently the latest supported major version is MEMCACHE_1_5. The minor version will be automatically
+          determined by our system based on the latest supported minor version.
+        default_value: :MEMCACHE_1_5
+        values:
+          - :MEMCACHE_1_5
+          - :MEMCACHE_VERSION_UNSPECIFIED
```

Comment:
There isn't a good reason why a user would set MEMCACHE_VERSION_UNSPECIFIED, so I'd recommend omitting it.

Reply:
fixed